pandas/_libs/src/ujson/lib/ultrajsondec.c

```diff
@@ -157,19 +157,18 @@ JSOBJ FASTCALL_MSVC decode_numeric(struct DecoderState *ds) {
             case '7':
             case '8':
             case '9': {
-                // FIXME: Check for arithmetic overflow here
-                // PERF: Don't do 64-bit arithmetic here unless we know we have
-                // to
-                intValue = intValue * 10ULL + (JSLONG)(chr - 48);
-
-                if (intValue > overflowLimit) {
-                    return SetError(ds, -1, overflowLimit == LLONG_MAX
-                                                ? "Value is too big"
-                                                : "Value is too small");
+                // PERF: Don't do 64-bit arithmetic here unless we have to
```

Comment:
In this case I think you can just vendor the entire switch block, no?

https://github.com/ultrajson/ultrajson/blob/6833495c6f4eae8ef20bc52aeaeb2c069f98ee3f/lib/ultrajsondec.c#L115

Reply:
I'm trying to get the tests finished and wrap-up this PR. I will then give it a try with the full lib update in a separate contribution. Btw, what's your recommendation on how to do that? Also through filing an issue?

Reply:
For user facing things you would normally open with an issue for discussion, but in this case since we would just be updating a vendored library you can just push a PR